src/CarbonAware.Tools/CarbonAware.Tools.WattTimeClient/src/Configuration/WattTimeClientConfiguration.cs

```diff
@@ -23,6 +23,12 @@ public class WattTimeClientConfiguration
     /// </summary>
     public string BaseUrl { get; set; } = "https://api2.watttime.org/v2/";
 
+    /// <summary>
+    /// Gets or sets the cached expiration time (in seconds) for a BalancingAuthority instance.
+    /// It defaults to 60 secs.
+    /// </summary>
+    public int BACacheExpirationInSecs { get; set; } = 60;
```

Comment:
After our chat, I was thinking about this, how about 5 minutes to correspond with the resolution that WattTime gives forecasts?
```suggestion
    public int BACacheExpirationInSecs { get; set; } = 300;
```